tests/Microsoft.Bot.Builder.TestBot.Tests/stylecop.json

```diff
@@ -0,0 +1,10 @@
+{
+  "settings": {
+    "namingRules": {
+      "allowCommonHungarianPrefixes": true,
+      "allowedHungarianPrefixes": [
+        "lu"
```

Comment:
> "lu" [](start = 7, length = 5)

ooooo can we add lg to this list?

Reply:
Will add it in the next PR (I got tired of stylecop complaining about Hungarian notation for lu).